sw/host/opentitanlib/src/dif/lc_ctrl.rs

```diff
@@ -107,6 +413,15 @@ impl DifLcCtrlToken {
             .for_each(|(word, out)| *out = word);
         out_words
     }
+
+    /// Returns 1 if the token is zero.
+    pub fn is_zero(self) -> bool {
+        let mut res = true;
+        for b in self.0.data {
+            res = res && (b == 0);
+        }
+        res
```

Comment:
```suggestion
        self.0.data.iter().all(|&x| x == 0)
```
or even
```suggestion
        self.0.data == [0; 16]
```

Reply:
Went for the latter one, since it seems more intuitive.